### DISCUSSION TITLE
Using AI to add necessary comments to the code

### DISCUSSION THREAD

**Discussion by kaixinol** [XusinboyBekchanov/VisualFBEditor#619] [Ideas]
I don't quite understand the source code of this IDE, so I want to write a Python tool to automatically add comments to a large amount of code in this project. 
Do you agree with me doing this?

**Comment by XusinboyBekchanov** [comment]
Yes, I don't mind, you can add a plugin

**Reply by kaixinol** [reply]
Um, what I mean is, to directly read the source code using Python and then add comments. 
I also thought about adding a native plugin, but I couldn't find the documentation

**Reply by XusinboyBekchanov** [reply]
Here is an example of how to add an Add-in.
https://github.com/XusinboyBekchanov/VisualFBEditor/blob/master/Examples/Add-In/My%20Add-In.bas

It doesn't matter what the tool is: Python Tool or FreeBasic Addin.
